src/CMakeLists.txt

```diff
@@ -1287,6 +1289,7 @@ add_custom_target(vstart DEPENDS
 add_custom_target(cephfs_testing DEPENDS
     vstart
     rados
+    cython_modules
```

Comment:
I'm not sure that `cephfs_testing` is the right place to put the only dependency on cython bindings. do you guys have a better idea of which targets will actually depend on `cython_rbd` and `cython_cephfs`?